src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/CollectSVEvidence.java

```diff
@@ -764,4 +847,160 @@ public VariantContext next() {
             return result;
         }
     }
+
+    @VisibleForTesting
+    final static class CountCounter {
+        private final int[] lowCounts;
+        private final SortedMap<Integer, Integer> highCounts;
+        private int nCounts;
+        private long totalCounts;
+
+        public CountCounter() {
+            lowCounts = new int[3000];
+            highCounts = new TreeMap<>();
+        }
+
+        public void addCount( final int count ) {
+            nCounts += 1;
+            totalCounts += count;
+            if ( count < lowCounts.length ) {
+                lowCounts[count] += 1;
+            } else {
+                highCounts.compute(count, (k,v) -> v==null ? 1 : v+1);
+            }
+        }
+
+        public int getNZeroCounts() {
+            return lowCounts[0];
+        }
+
+        public int getMeanCount() {
+            return (int)Math.round((double)totalCounts/nCounts);
+        }
```

Comment:
```suggestion
        public double getMeanCount() {
            return Math.round((double)totalCounts/nCounts);
        }
```
I think most users would expect an exact mean, rounding to 2 decimal places would be fine though